build.gradle

```diff
@@ -81,54 +81,112 @@ task buildSconsArtifacts {
         }
     }
 
+    return sconsExecutableFile
+}
+
+def createSconsTasks(File sconsExecutableFile, boolean clean) {
+    def defaultArgs = [
+        "--directory=.",
+        "platform=android",
```

Comment:
Based on the logic below, seems like `platform=android` should be removed from the `defaultArgs`.

Reply:
Ah, thanks! Fixed in my latest pushed